### ISSUE TITLE
BEER whitelist on EQUALIZER, send LOGO

### ISSUE THREAD

**Issue by Mikerano** [ftm1337/crypto-icons#3]
![0x94Ba6241BC7E6F7F24a5B95fE6726B31C74881aB](https://github.com/ftm1337/crypto-icons/assets/36679633/99b9dfdc-ecba-48d3-88cd-643c38605916)

**Comment by Mikerano** [comment]
Yeah here Logo for Whitlist, liq done, looking forward - if any question please ask me on Discord Thanx

**Comment by ftm1337** [comment]
hey @Mikerano, can you please remove the outer white box please, and make it Transparent

**Comment by Mikerano** [comment]
![0x94Ba6241BC7E6F7F24a5B95fE6726B31C74881aB](https://github.com/ftm1337/crypto-icons/assets/36679633/70319ba8-bd05-4b0c-a807-d2622ebf385d)

**Comment by Mikerano** [comment]
hello here is like you want i hope at least, please tell me if it is ok

**Comment by Mikerano** [comment]
![image](https://github.com/ftm1337/crypto-icons/assets/36679633/92daa313-ec89-4886-b129-7c34f9c7fa4e)
![image](https://github.com/ftm1337/crypto-icons/assets/36679633/8d95670c-8a31-4f58-bc17-3f9f135c490a)

**Comment by ftm1337** [comment]
@Mikerano looks good! will add

**Comment by Mikerano** [comment]
thank you - when can we be whitelisted?

**Comment by Mikerano** [comment]
Happy Easter - how is our status quo? i see the pic but staking not possilbe rn ty

**Comment by ftm1337** [comment]
Happy Easter, @Mikerano :rabbit: 
Team is away for a few days due to the holidays, so probably later in the week..

**Comment by Mikerano** [comment]
hey good afternoon - do you know more about wl already? haha we wanna bribe stake and vote
